src/api/coingecko.js

```diff
@@ -0,0 +1,16 @@
+const CoinGecko = require('coingecko-api');
```

Comment:
Do we really need this module? redstone-api can do everything that is needed here even better ;)